### ISSUE TITLE
Remove dead code before public preview

### ISSUE THREAD

**Issue by CodeMonkeyLeet** [openenclave/openenclave#57]
Tracking issue to accumulate known cleanup prior to public preview:
- [x] 3rdparty/libunwind/libunwind/aux-aux is omitted in build (see #56)
- [x] 3rdparty/musl/wrap (superseded by cmake/wrap.cmake)
- ~~prereqs/packages (consider converging intel dependencies with scripts/install-prereqs)~~ (leaving this in and separated from install-prereqs until AESM support is removed)
- [x] tools/bin appears defunct
- ~~tests/*/enc/private.pem and sign.conf (convert to using properties macro after #160)~~ (tracked separately as #465)
- [x] tests/eenter is not building or running
- [x] tests/hashstr is not building or running
- [x] tests/measure is not building or running
- [x] tests/SampleApp/SampleApp/signconf.xml (redundant, replaced by sign.conf)
- [x] tests/pingpong-shared/enc/signconf.xml (redundant, replaced by sign.conf)
- [x] tests/pingpong/enc/signconf.xml (redundant, replaced by sign.conf)
- [x] tests/*/gdb.command files (inconsistently included, do not leverage oe-gdb)
